solidity/contracts/ERC20NoData.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.0;
+
+import '@openzeppelin/contracts/token/ERC20/ERC20.sol';
+import '@openzeppelin/contracts/utils/Context.sol';
+import '@openzeppelin/contracts/access/Ownable.sol';
+
+/**
+    @dev Basic mintable+burnable form of ERC20.
+*/
+contract ERC20NoData is Context, Ownable, ERC20 {
+    constructor(string memory name, string memory symbol) ERC20(name, symbol) {}
+
+    function mintNoData(address to, uint256 amount) external onlyOwner {
```

Comment:
I assume we want to allow the plugin implementation to be able to work with as many existing contracts as possible. If so, we should rename these functions to the standard implementations you find in popular libraries like OpenZeppelin, so instead of `mintNotData`, use `mint`, etc.

Same applies to the ERC721 and the resulted ABI JSON